### DISCUSSION TITLE
Improvement regarding the "Deploy Snapshot" terminology

### DISCUSSION THREAD

**Discussion by JmStawikowski** [FlowFuse/flowfuse#4581] [General]
Hello,

As a future FlowFuse user, I would like to suggest an improvement regarding the "Deploy Snapshot" terminology:

**Issue**: The term "Deploy" can be misleading, as it suggests deployment to external devices, such as hardware ("devices").

**Suggestion**: Replace it with "Restore Snapshot", "Apply Snapshot", or "Revert to Snapshot".

**Benefits**:
* Increased clarity for users
* Better representation of the actual action performed

This small change could significantly improve the user experience.

Best regards,
![Deploy Snapshot](https://github.com/user-attachments/assets/af1108c2-388a-4560-9d13-48cf9df31215)

**Comment by joepavitt** [comment]
@cstns can we action this please? I like "Restore Snapshot"

**Reply by cstns** [reply]
Created https://github.com/FlowFuse/flowfuse/issues/4583

**Comment by joepavitt** [comment]
@JmStawikowski thanks for the suggestion on this, just to confirm that @cstns has just deployed your recommendation, it should be live on FlowFuse Cloud in the next 5-10 minutes.
